src/credentials/attestation_verifier/DeviceAttestationVerifier.cpp

```diff
@@ -111,5 +113,30 @@ void SetDeviceAttestationVerifier(DeviceAttestationVerifier * verifier)
     gDacVerifier = verifier;
 }
 
+static inline Platform::ScopedMemoryBufferWithSize<uint8_t> CopyByteSpanHelper(const ByteSpan & span_to_copy)
+{
+    Platform::ScopedMemoryBufferWithSize<uint8_t> bufferCopy;
+    bufferCopy.Alloc(span_to_copy.size());
+    memcpy(bufferCopy.Get(), span_to_copy.data(), span_to_copy.size());
```

Comment:
```suggestion
    if (bufferCopy.Alloc(span_to_copy.size()))
    {
        memcpy(bufferCopy.Get(), span_to_copy.data(), span_to_copy.size());
    }
```
And as a follow-up (separate PR) we should think about how to handle allocation failure here....